### PR TITLE
Add missing dispose calls.

### PR DESCRIPTION
**Pull request by Aenima4six2**

### CODE REVIEW

File: Source/EasyGelf.Core/Transports/BufferedTransport.cs [hunk 1]
```diff
@@ -60,6 +65,9 @@ public void Close()
             buffer.CompleteAdding();
             cancellationTokenSource.Cancel();
             stopEvent.WaitOne();
+            
+            buffer.Dispose();
```
Comment (Aenima4six2) [inline Source/EasyGelf.Core/Transports/BufferedTransport.cs]: Probably don't matter much but better to be safe.

**Comment by Pliner** [comment]
Thanks for contribution!

@Aenima4six2 Could you also bump version in appveyour.yml?

File: Source/EasyGelf.Log4Net/GelfTcpAppender.cs [hunk 2]
```diff
@@ -10,7 +8,7 @@ public sealed class GelfTcpAppender : GelfAppenderBase
     {
         public GelfTcpAppender()
         {
-            TcpTransportConfiguration defaultCfg = TcpTransportConfiguration.GetDefaultConfiguration();
+            var defaultCfg = TcpTransportConfiguration.GetDefaultConfiguration();
```
Comment (energister) [inline Source/EasyGelf.Log4Net/GelfTcpAppender.cs]: IMHO it's more comfortable to know with which type you are dealing with. So I don't like to use var when right part of the assignment doesn't specify the type explicitly

Comment (Aenima4six2) [inline Source/EasyGelf.Log4Net/GelfTcpAppender.cs]: Eh, its not worth the bloat and loss of readability to me. Any modern text editor or IDE can show you the compiler inferred type with a simple mouse hover. Its also rather difficult for me to memorize the particulars of a type or contract when working across many projects, so I regularly rely on my tools to provide that resolution when needed.

**Pull request closed without merging**